recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,4 @@
+MACOSX_DEPLOYMENT_TARGET:   # [osx]
+- '10.12'                   # [osx]
+CONDA_BUILD_SYSROOT:        # [osx]
+- /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.12.sdk  # [osx]
```

Comment:
Remove the 2 entries above and do a rerender. `conda-forge-ci-setup` will add the correct path.